test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval_long_expiration.test

```diff
@@ -72,7 +71,7 @@ payload='<?xml version="1.0"?>
     <attribute>temperature</attribute>
   </attributeList>
   <reference>http://127.0.0.1:'${LISTENER_PORT}'/notify</reference>
-  <duration>P99Y</duration>
+  <duration>PT11S</duration>
```

Comment:
I understand that the valgrind problems use to happen due to things run slowly under valgrind. Thus, I don't understand why reducing duration make valgrind work (it should be the opposite, shouldn't it?)...

Anyway, valgrind "adaptations" shouldn't go agains the test case purpose and, in this case, the purpose is to test with very long subscriptions as it was before the modifications.